factor/sms/classes/local/smsgateway/aws_sns.php

```diff
@@ -78,10 +80,25 @@ public function send_sms_message(string $messagecontent, string $phonenumber) :
 
         // Actually send the message.
         try {
-            $client->publish([
+            $result = $client->publish([
                 'Message' => $messagecontent,
                 'PhoneNumber' => $phonenumber,
             ]);
+
+            $data = array(
+                'relateduserid' => null,
+                'context' => \context_user::instance($USER->id),
+                'other' => array(
+                    'userid' => $USER->id,
+                    'debug' => array(
+                        'phonenumber' => $phonenumber,
```

Comment:
I think we should likely redact the phone number. Logs are generally sensitive but I dont think there is a reason to keep it in, and its 1 less piece of sensitive information.

https://github.com/catalyst/moodle-tool_mfa/blob/01635da474c4b863374de215dbccaacc2caacd32/factor/sms/classes/factor.php#L235 This could be moved to a function in the helper. Not mission critical however